sys/src/9/amd64/vq9p.c

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
Not x86 specific in the code, but my understanding was that virtio is mainly available and usable on x86 platforms (s390 which is also mentiond in the spec left alone). Plus, it relies on pci, and pci.c is in 386, though devpci.c is in port. That made me think about intel-specificity of virtio-pci. I just cannot see anything like pci.c in aarch64...